tools/test-runner-cli.ts

```diff
@@ -19,10 +19,9 @@ const args = parseTestRunnerCliArgs(process.argv.slice(2));
 logger.verbose('TestRunnerCli.Init.Config', inspect(args));
 
 const TEST_ROOT = path.join(__dirname, '..', 'test');
-const TEST_DATA_ROOT = path.join(__dirname, '..', 'deps', 'data', 'data', 'test');
-const TEST_DATA_NODE_ROOT = path.join(TEST_DATA_ROOT, 'node');
-const TEST_DATA_ONNX_ROOT = path.join(TEST_DATA_ROOT, 'onnx', 'v7');
-const TEST_DATA_OP_ROOT = path.join(TEST_DATA_ROOT, 'ops');
+const TEST_DATA_NODE_ROOT = path.join(__dirname, '..', 'deps/onnx/onnx/backend/test/data/node');
+const TEST_DATA_ONNX_ROOT = path.join(__dirname, '..', 'deps/data/data/test/onnx/v7');
```

Comment:
I think this can be TEST_DATA_MODEL_ROOT or  TEST_DATA_ONNX_MODEL_ROOT. The naming is little confusing.